src/components/content/lazy.tsx

```diff
@@ -21,7 +32,12 @@ export function Lazy(props: LazyProps) {
 
   // disable lazy load if content-only and probably within iframe
   if (router.pathname.startsWith('/content-only/')) {
-    return <>{props.children}</>
+    return (
+      <>
+        <LazyIframeResizer />
```

Comment:
can you put this into `src/pages/content-only/[...slug].tsx`?

this place is only a fix for another problem and not directly related to the content-api. Adding it in the route is more reliable